x-pack/plugins/security/server/routes/authentication/common.ts

```diff
@@ -36,124 +35,67 @@ export function defineCommonRoutes({
   buildFlavor,
   docLinks,
 }: RouteDefinitionParams) {
-  // Generate two identical routes with new and deprecated URL and issue a warning if route with deprecated URL is ever used.
-  // For a serverless build, do not register deprecated versioned routes
-  for (const path of [
-    '/api/security/logout',
-    ...(buildFlavor !== 'serverless' ? ['/api/security/v1/logout'] : []),
-  ]) {
-    const isDeprecated = path === '/api/security/v1/logout';
-    router.get(
-      {
-        path,
-        security: {
-          authz: {
-            enabled: false,
-            reason: 'This route must remain accessible to 3rd-party IdPs',
-          },
-        },
-        // Allow unknown query parameters as this endpoint can be hit by the 3rd-party with any
-        // set of query string parameters (e.g. SAML/OIDC logout request/response parameters).
-        validate: { query: schema.object({}, { unknowns: 'allow' }) },
-        options: {
-          access: 'public',
-          excludeFromOAS: true,
-          authRequired: false,
-          tags: [ROUTE_TAG_CAN_REDIRECT, ROUTE_TAG_AUTH_FLOW],
-          ...(isDeprecated && {
-            deprecated: {
-              documentationUrl: docLinks.links.security.deprecatedV1Endpoints,
-              severity: 'warning',
-              message: i18n.translate('xpack.security.deprecations.logoutRouteMessage', {
-                defaultMessage:
-                  'The "{path}" URL is deprecated and will be removed in the next major version. Use "/api/security/logout" instead.',
-                values: { path },
-              }),
-              reason: {
-                type: 'migrate',
-                newApiMethod: 'GET',
-                newApiPath: '/api/security/logout',
-              },
-            },
-          }),
+  router.get(
+    {
+      path: '/api/security/logout',
+      security: {
+        authz: {
+          enabled: false,
+          reason: 'This route must remain accessible to 3rd-party IdPs',
         },
       },
-      async (context, request, response) => {
-        const serverBasePath = basePath.serverBasePath;
-        if (isDeprecated) {
-          logger.warn(
-            `The "${serverBasePath}${path}" URL is deprecated and will stop working in the next major version. Use "${serverBasePath}/api/security/logout" URL instead.`,
-            { tags: ['deprecation'] }
-          );
-        }
+      // Allow unknown query parameters as this endpoint can be hit by the 3rd-party with any
+      // set of query string parameters (e.g. SAML/OIDC logout request/response parameters).
+      validate: { query: schema.object({}, { unknowns: 'allow' }) },
+      options: {
+        access: 'public',
+        excludeFromOAS: true,
+        authRequired: false,
```

Comment:
__question:__ while you're here, would you mind replacing deprecated `authRequired` with `security.authc.enabled` in all routes in this file?